000-vip-init.php

```diff
@@ -212,6 +216,9 @@
 	require_once( __DIR__ . '/search/search.php' );
 }
 
+
+
+
```

Comment:
Not sure we need to add those line breaks.

Reply:
Yeah, a leftover from moving the lines. Not sure why linter didn't catch it.